app/org/sagebionetworks/bridge/stormpath/StormpathDirectoryDao.java

```diff
@@ -177,9 +177,9 @@ private static AccountStoreMapping getApplicationMapping(String href, Applicatio
     private void adjustPasswordPolicies(Study study, Directory directory) {
         PasswordPolicy passwordPolicy = directory.getPasswordPolicy();
         
-        // According to the documentation, there is only one...
-        ModeledEmailTemplate template = passwordPolicy.getResetEmailTemplates().iterator().next();
-        updateTemplate(study, template, study.getResetPasswordTemplate(), "resetPassword");
+        for (ModeledEmailTemplate template : passwordPolicy.getResetEmailTemplates()) {
+            updateTemplate(study, template, study.getResetPasswordTemplate(), "resetPassword");
+        }
```

Comment:
In StormpathDirectoryDaoTest, we have asserts that there's only 1 reset password template and only 1 account verification email template.

This is cause for concern because (a) it means the fix we tried to make doesn't match up with the outcome we're seeing and (b) the unit tests don't test the change that we made.